bridge/bridge/WorkflowUtils.py

```diff
@@ -100,3 +105,44 @@ def create_conversion_rate_calculator_factory(execution_context, databases, nati
 	return ConversionRateCalculatorFactory(databases, native_facade.extract_mosaic_id().formatted, execution_context.is_unwrap_mode)
 
 # endregion
+
+
+# region prepare_send
+
+def prepare_send(network, request, conversion_function, fee_multiplier):
+	"""Performs basic calculations and validation prior to sending a payout transaction."""
+
+	def make_error(error_message):
+		return PrepareSendResult(error_message, None, None)
+
+	if not fee_multiplier:
+		fee_multiplier = Decimal('1')
+	else:
+		fee_multiplier *= Decimal(conversion_function(10 ** 12)) / Decimal(10 ** 12)
+
+	transfer_amount = conversion_function(request.amount)
+
+	max_transfer_amount = int(network.config.extensions.get('max_transfer_amount', 0))
+	if max_transfer_amount and transfer_amount > max_transfer_amount:
+		return make_error(f'gross transfer amount {transfer_amount} exceeds max transfer amount {max_transfer_amount}')
+
+	return PrepareSendResult(None, fee_multiplier, transfer_amount)
+
+# endregion
+
+
+# region check_expiry
+
+def check_expiry(config_extensions, database, request):
+	"""Determines if the specified request is expired (timed out)."""
+
+	request_lifetime_hours = int(config_extensions.get('request_lifetime_hours', 0))
```

Comment:
Not sure if the lack of `request_lifetime_hours` in config or else its explicit value to zero should be treated either as infinite lifetime or zero lifetime. In the latter case every single transaction should fail.

Reply:
i think infinite is fine (original behavior), also having setting like this "every single transaction should fail" probably doesn't make much sense.